tests/ha/test_cluster_shutdown_start.py

```diff
@@ -84,7 +84,7 @@ def setup_class(cls):
         cls.s3acc_name = cls.s3acc_email = cls.bucket_name = cls.object_name = None
         cls.multipart_obj_path = None
         cls.mgnt_ops = ManagementOPs()
-        cls.system_random = random.SystemRandom()
+        cls.system_random = secrets.SystemRandom()
```

Comment:
This should be random.SystemRandom only right? As we want to generate randomness.